editor/project_manager.h

```diff
@@ -93,42 +93,41 @@ class ProjectDialog : public ConfirmationDialog {
 
 	EditorFileDialog *fdialog = nullptr;
 	EditorFileDialog *fdialog_install = nullptr;
-	AcceptDialog *dialog_error = nullptr;
 
 	String zip_path;
 	String zip_title;
-	String fav_dir;
 
-	String created_folder_path;
+	void _set_message(const String &p_msg, MessageType p_type = MESSAGE_SUCCESS, InputType p_input_type = PROJECT_PATH);
+
+	String _validate_path();
 
-	void _set_message(const String &p_msg, MessageType p_type = MESSAGE_SUCCESS, InputType input_type = PROJECT_PATH);
+	String _last_target_auto_dir;
```

Comment:
I'd keep this along with the other variables, also let's keep the naming consistent and remove the `_`, this file doesn't use that for member variables (even though some files do)